libs/sdk/testing/src/lib/matchers/matchers.spec.ts

```diff
@@ -135,9 +135,11 @@ describe('Jasmine matchers', () => {
     try {
       expect(elem).toHaveCssClass('.with-dot');
     } catch (err) {
-      expect(err.message).toEqual(
-        'Please remove the leading dot from your class name.'
-      );
+      if (err instanceof Error) {
```

Comment:
I know this isn't your fault, but could you put a `fail('Expected test to fail.')` just before the `catch` block?

Reply:
Done